docker-compose.yml

```diff
@@ -134,6 +118,190 @@ services:
         condition: service_healthy
     volumes:
       - ./rabbitmq/consumers/stock:/HOME/FLASK-APP
+  ## Router
+  router01:
```

Comment:
Could you change the naming of the parts a bit to make it clearer for which service the elements are running? I opted for a prefix `<service>_router01` etc. The same goes for the container names. This would require a bit of tweaking in the config files as well.